.github/actions/run_tests/action.yml

```diff
@@ -49,5 +49,4 @@ runs:
         coverage html
         coverage report
         coverage-badge -o htmlcov/coverage.svg
```

Comment:
We should be able to remove the `coverage run --rcfile=${{ inputs.coverage_config }} testing_main.py` call (and the corresponding calls to `coverage`, since `pytest` already picks up all "old" tests.